src/server/pmix_server.c

```diff
@@ -290,6 +293,8 @@ PMIX_EXPORT pmix_status_t PMIx_server_init(pmix_server_module_t *module,
         PMIX_RELEASE_THREAD(&pmix_global_lock);
         return PMIX_ERR_INIT;
     }
+    pmix_perrmgr.start();
+    //pmix_pdetector.start();
```

Comment:
Why is this commented out ? Who starts the detector ?

Reply:
This detector is a new one in pmix which should be based on a P2P communication(for heartbeat), but pmix doesn't have one. I write this one ahead.  The working one is in orte.

Reply:
For the output, i will replace all pmix_output.
Also some people use pmix_output_verbose, others use PMIX_OUTPUT_VERBOSE, i want to make sure which one do you prefer ?

Reply:
It''
s not really a matter of preference. PMIX_OUTPUT_VERBOSE gets compiled out in a release build, when the others are always available. Thus, everything that is at the level where only a developer can understand the intricacies should be PMIX_OUTPUT_VERBOSE, and pmix_output_verbose should be used for everything else.